addon/components/nrg-home-cards/home-card/component.js

```diff
@@ -14,22 +14,28 @@ export default Component.extend({
 
   router: service(),
 
-  visible: computed('currentUser.roles', function() {
-    const role = this.get('role');
-    if (role) {
-      return this.get('currentUser.content').hasRole(role);
+  visible: computed('currentUser.roles', 'role', function() {
+    if (!this.role) {
+      return true;
+    }
+    const roles = Array.isArray(this.role) ? this.role : [this.role];
+    const needsAllRoles = this.needsAllRoles;
+    const currentUserContent = this.get('currentUser.content');
+    if (needsAllRoles) {
+      return roles.every(role => currentUserContent.hasRole(role));
+    } else {
+      return roles.some(role => currentUserContent.hasRole(role));
     }
```

Comment:
The `else` block isn't really necessary since you return from the `if` block.
```suggestion
    }
    return roles.some(role => currentUserContent.hasRole(role));
```